s3_management/manage.py

```diff
@@ -190,15 +191,15 @@ def gen_file_list(
             else self.objects
         )
         subdir = self._resolve_subdir(subdir) + '/'
-        for obj in objects:
+        for obj, checksum in objects.items():
             if package_name is not None:
                 if self.obj_to_package_name(obj) != package_name:
                     continue
             if self.is_obj_at_root(obj) or obj.startswith(subdir):
-                yield obj
+                yield obj, checksum
```

Comment:
The return type of this function needs updating

Reply:
I believe this is fixed now.